src/Factories/TestCaseFactory.php

```diff
@@ -152,15 +152,23 @@ public function build(TestSuite $testSuite): array
      */
     public function makeClassFromFilename(string $filename): string
     {
+        if ('\\' === DIRECTORY_SEPARATOR) {
```

Comment:
This will only happen on windows?

Reply:
Might be better to just use `PHP_OS_FAMILY === 'Windows'`?

Reply:
Yep this can happen only on Windows, that's a fairly common case 🙂